src/StylePlugin.js

```diff
@@ -104,8 +104,19 @@ function applyStyles(state, tr) {
 
     tr.doc.descendants(function (child, pos) {
         const contentLen = child.content.size;
+		// [FS] IRAD-1170 2021-02-02
+		// FIX: When loading some documents on load show "Cannot read nodeSize property of undefined" error.
+		const docLen = tr.doc.content.size;
+		let end = pos + contentLen;
```

Comment:
Whenever haveEligibleChildren returns false, the calculation of 'end' being done here is thrown away.  The fix would perform better if this was done after you determine you're going to use the value.